org.mixedrealitytoolkit.input/Tests/Runtime/FuzzyGazeInteractorTests.cs

```diff
@@ -229,6 +240,17 @@ public IEnumerator GazeFallbackWhenEyeGazeIsDisabledTest()
         {
             // Confirm a FuzzyGazeInteractor is active in the scene
             FuzzyGazeInteractor fuzzyGazeInteractor = FindObjectUtility.FindFirstObjectByType<FuzzyGazeInteractor>();
+
+            #region TrackedPoseDriverWithFallback disabling
+            // Disable the TrackedPoseDriverWithFallback component that was added for XRI3 migration.  Disable it to ensure that
+            // the pre-XRI3 ActionBasedControllerWithFallbacks functionality is properly tested.
+            TrackedPoseDriverWithFallback trackedPoseDriverWithFallback = fuzzyGazeInteractor.GetComponent<TrackedPoseDriverWithFallback>();
+            if (trackedPoseDriverWithFallback != null)
+            {
```

Comment:
I still don't understand why this is needed, if we're not change the older prefab.   Since the older prefab is untouched, wouldn't the old tests continue to work?

Reply:
You are right, I was under the wrong impression that the old prefabs would be changing.  Removed in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/746/commits/5c757d880deef6cb6375b82340f815012616c1e9 , resolving.